app/routes/cdo/manage/live.js

```diff
@@ -0,0 +1,34 @@
+const { routes, views } = require('../../../constants/cdo/index')
+const { anyLoggedInUser } = require('../../../auth/permissions')
+const { addBackNavigation } = require('../../../lib/back-helpers')
+const ViewModel = require('../../../models/cdo/manage/live')
+const { getLiveCdos, getLiveCdosWithinMonth } = require('../../../api/ddi-index-api/cdos')
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${routes.manage.get}/{tab?}`,
+    options: {
+      auth: { scope: anyLoggedInUser },
+      handler: async (request, h) => {
+        const backNav = addBackNavigation(request)
```

Comment:
You probably don't need the intelligent back nav since, presumably, back will take you to the hard-coded link of home page. However, I;'m not sure if nav from the live CDOs page needs the src param to navigate back to live CDOs etc. Just flagging this as something to check

Reply:
You're right, it's not needed, there is no back nav on the prototype - which means something is missing from my design